src/coreComponents/common/logger/ErrorHandling.hpp

```diff
@@ -0,0 +1,254 @@
+/*
+ * ------------------------------------------------------------------------------------------------------------
+ * SPDX-License-Identifier: LGPL-2.1-only
+ *
+ * Copyright (c) 2016-2024 Lawrence Livermore National Security LLC
+ * Copyright (c) 2018-2024 TotalEnergies
+ * Copyright (c) 2018-2024 The Board of Trustees of the Leland Stanford Junior University
+ * Copyright (c) 2023-2024 Chevron
+ * Copyright (c) 2019-     GEOS/GEOSX Contributors
+ * All rights reserved
+ *
+ * See top level LICENSE, COPYRIGHT, CONTRIBUTORS, NOTICE, and ACKNOWLEDGEMENTS files for details.
+ * ------------------------------------------------------------------------------------------------------------
+ */
+
+/**
+ * @file ErrorHandling.hpp
+ */
+
+#ifndef INITIALIZATION_ERROR_LOGGER_HPP
+#define INITIALIZATION_ERROR_LOGGER_HPP
+
+#include "common/DataTypes.hpp"
+
+namespace geos
+{
+
+/**
+ * @class ErrorLogger
+ * @brief Class to format and write the error/warning message that occured during the initialization
+ */
+class ErrorLogger
+{
+
+public:
+
+  /**
+   * @brief Construct a new Error Logger object
+   *
+   */
```

Comment:
Please remove those empty comment lines (here and below):
```suggestion
  /**
   * @brief Construct a new Error Logger object
   */
```